.travis.yml

```diff
@@ -0,0 +1,15 @@
+git:
+  depth: 1
+dist: xenial
+languages: python
+services:
+  - docker
+before_install:
+  - pip install --upgrade pip --user
+  - pip install molecule docker --user
+script:
+  - molecule --version
+  - ansible --version
+  - molecule test
+  #- docker build --tag redhat-service-assurance/smart-gateway-operator:latest .
```

Comment:
You can just delete this line and the one below it since we're going to automatically build and deploy on merges to master.

Reply:
'molecule test' runs the playbooks, are you referring to the commented and '--version' lines?

Reply:
Not above... below :)

```
#- docker build --tag redhat-service-assurance/smart-gateway-operator:latest .
#- docker images
```